docs/latest/user/rate-limit.md

```diff
@@ -591,6 +601,23 @@ data:
           replicas: 1
           container:
             image: envoyproxy/ratelimit:master
+            env:
+            - name: REDIS_SOCKET_TYPE
```

Comment:
imo there is no value in adding the same env again here, these should be different hence I pointed out the `CACHE_KEY_PREFIX` env earlier